cli/src/utils/proverUtils.ts

```diff
@@ -1,4 +1,4 @@
-import { getProverNameByArch } from "./initTestEnv";
+import { getProverNameByArch } from "./process";
```

Comment:
can we rename ``process.ts`` to something more meaningful?
I am also seeing that ``process.ts`` combines functionality to start the test-validator and the prover server.
To get a more meaningful name maybe it makes sense to split the functionality into two files like:
1. ``process-start-test-validator.ts``
2. ``process-start-prover-server.ts`` 

or move the prover server part into `` prover-utils.ts``  and rename only ``process.ts`` to ``process-start-test-validator.ts``